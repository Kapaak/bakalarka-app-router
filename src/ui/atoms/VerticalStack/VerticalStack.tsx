import { PropsWithChildren } from "react";

import { ClassName } from "@/domains";

interface VerticalStackProps {
  className?: ClassName;
}

export const VerticalStack = ({
  children,
  className,
}: PropsWithChildren<VerticalStackProps>) => {
  return <div className={`flex flex-col ${className}`}>{children}</div>;
};
