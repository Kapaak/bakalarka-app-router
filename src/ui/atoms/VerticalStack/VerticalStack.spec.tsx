import { render } from "@testing-library/react";

import { VerticalStack } from "./VerticalStack";

describe("VerticalStack", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<VerticalStack />);

    expect(baseElement).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
});
