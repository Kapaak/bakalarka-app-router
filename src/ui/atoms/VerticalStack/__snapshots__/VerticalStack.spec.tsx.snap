// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerticalStack should render successfully 1`] = `
<body>
  <div>
    <div
      class="flex flex-col undefined"
    />
  </div>
</body>
`;
